packages/artillery-engine-playwright/index.js

```diff
@@ -186,10 +203,15 @@ class PlaywrightEngine {
         const fn =
           self.processor[spec.testFunction] ||
           self.processor[spec.flowFunction];
+
         await fn(page, initialContext, events);
 
+        // After the user flow function has finished we will have all the performance entries needed to set the spans
+        const tracePerformanceEntries = self.processor[spec.traceFlowFunction]
+        tracePerformanceEntries(entries, spec.name)
```

Comment:
shouldn't this be called only when `self.tracing` is available too?

Reply:
Oh, missed that! good catch🙌🏻